src/ecs/resources/input.rs

```diff
@@ -51,22 +89,33 @@ impl InputHandler {
                             // second `Pressed` event.
                         }
                         Entry::Vacant(entry) => {
-                            entry.insert(KeyQueryState::Queried);
+                            entry.insert(KeyQueryState::NotQueried);
```

Comment:
I think this change should be cherry picked out as another commit. We should try to be clear on what our commits are doing, adding mouse button events != bug fixes